drafts/draft-0005.md

```diff
@@ -0,0 +1,138 @@
+## Preamble
+
+```
+SEP: <to be assigned>
+Title: Data Entry Namespaces
+Author: Mister.Ticot <mister.ticot@cosmic.plus>
+Status: Draft
+Created: 2018-10-26
+```
+
+## Simple Summary
+
+Defines a standard syntax to represent namespaces in data entries.
+
+## Abstract
+
+This SEP provides a standard way to represent
+[namespaces](https://en.wikipedia.org/wiki/Namespace) into Stellar accounts
+data entries, and provides guideline about how to parse them and to handle
+edge cases.
+
+## Motivation
+
+As a generic mechanism to store key/value pairs on the ledger, data entries
+accept a very wide range of use cases. If each actor use it following its own
+syntax, it will lead to a confusing experience.
+
+Introducing namespaces allow to hierarchize and organize data, in such a way 
```

Comment:
What's the benefit to being able to turn it into a JSON structure? Is there a specific example you can show as to how this improves things over using the keys directly, besides just having a tidy looking data object?

Reply:
(answer in next item)